client/landing/gutenboarding/stepper-onboarding/steps.ts

```diff
@@ -0,0 +1,6 @@
+// this a list of all the steps, it's meant to list them, not preserve their order
+export const Steps = {
+	FirstStep: 'first',
+	SecondStep: 'second',
+	ThirdStep: 'third',
```

Comment:
```suggestion
	First: 'first',
	Second: 'second',
	Third: 'third',
```

Maybe? 

Also, any thoughts on what the intended structure will be for a Step? Is this something that will evolve beyond a primitive/string?